Jenkinsfile

```diff
@@ -39,6 +39,7 @@
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@use-mock") _
```

Comment:
Currently this should not depend on that PR branch.

Reply:
This line needs to be reverted once daos-stack/pipeline-lib#83 lands.  Or is there something I am not seeing?

Reply:
I do not see any changes in the pipeline-lib@use-mock branch that this PR depends on.
This repository does not use the pipeline-lib jenkinsfile or the packageBuildingPipeline.groovy.

Reply:
Fair enough.  It needs to be removed before it can be landed in any case.